server/public/layout.css

```diff
@@ -7,6 +7,11 @@
   font-size: 0.75rem;
 }
 
+.col.d-flex {
+  max-width: 90%;
+  margin: auto;
+}
+
```

Comment:
I wouldn't recommend directly overwriting what the bootstrap classes do, this can cause issues in other parts of your app that use the same classes. It would be better to make your own class that does this.

Reply:
Ok, that makes sense.  Thank you Scott